[HackerNewsUserThreads] New bridge

rakoo opened the pull request:
New bridge for HN's threads page of a specific user, for example for [my user](https://news.ycombinator.com/threads?id=rakoo) you get entries like this:

```
<entry>
  <title type="html">rakoo | on Our Year in Review: How We’ve Made Firefox Faster ...</title>
  <published>2020-12-18T14:40:56+00:00</published>
  <updated>2020-12-18T14:40:56+00:00</updated>
  <id>https://news.ycombinator.com/item?id=25462604</id>
  <link rel="alternate" type="text/html" href="https://news.ycombinator.com/item?id=25462604"/>
  <author>
    <name>rakoo</name>
  </author>
  <content type="html">
The concern with Pocket is primarily, IMO, that the service wasn&#x27;t and still isn&#x27;t Open Source, which is a hard sell for a browser that champions user control.
  </content>
</entry>
```

My goal is to know when there's been a reply to one of the comments I did, or anywhere in the chain (ie a reply of a reply, ...). HN doesn't provide any indications like this, so the next best thing is to use RSS :)

Commit:
[HackerNewsUserThreadsBridge] Add bridge for comments on threads of a user

Commit:
[HackerNewsUserThreadsBridge] Use better title and timestamp, add author

em92 commented:
Hi, @rakoo ! Please fix linting errors from Travis CI.
Everything else looks fine

rakoo commented:
Hey! Sorry I was a bit crowded lately, as you can imagine.

I've fixed the issue and modified the previous commit, it seems that the job hasn't restarted. Should I create a new PR ?

em92 commented:
No, that looks like a problem with Travis CI again. I will migrate to Github Actions and will try to start job on my own.

em92 committed:
dummy fix to turn on github actions

em92 commented:
gj @rakoo !
